admin/views/school/pupil-register.ejs

```diff
@@ -52,9 +60,12 @@
                     <td<% if (highlight && highlight.has(p.urlSlug.toString())) { %> class="highlight-item"<% } %>>
                         <div class="highlight-wrapper">
                             <% if (isFeatureEnabled('pupilEdit')) { %>
+                                <span class="bold-small truncate"><%= `${p.lastName}, ${p.foreName}` %></span>
+                                <!--
```

Comment:
Does this need to be commented out?  Does the feature toggling not do enough?

Reply:
Same as above, feature toggling is enough.